spec/controllers/questions_controller_spec.rb

```diff
@@ -105,10 +106,9 @@
       end
 
       xit "changes the question's attributes" do
```

Comment:
Seeing a lot of pending tests still.  I was reading up on CircleCI and one of the caveats with pushing up code that doesn't have an associated test is that we risk pushing up code that will break our site.  During checkin today, let's decide as a group how to proceed with future code that don't have tests.